pkg/scheduler/core/util.go

```diff
@@ -17,6 +17,14 @@ import (
 
 type calculator func([]*clusterv1alpha1.Cluster, *workv1alpha2.ResourceBindingSpec) []workv1alpha2.TargetCluster
 
+type UnschedulableError struct {
```

Comment:
It's better to put this struct to types.go

Reply:
which `types.go`?